mlrun/datastore/targets.py

```diff
@@ -154,11 +154,13 @@ def __init__(
         path=None,
         attributes: typing.Dict[str, str] = None,
         after_state=None,
+        columns_override=None,
     ):
         self.name = name
         self.path = str(path) if path is not None else None
         self.after_state = after_state
         self.attributes = attributes or {}
+        self.columns_override = columns_override or []
```

Comment:
should be `columns_override` (w/o the override)

Reply:
Done